AirCasting/Utils/Keychain/KeychainManager.swift

```diff
@@ -0,0 +1,51 @@
+// Created by Lunar on 16/08/2022.
+//
+
+import Foundation
+
+class KeychainManager {
+    enum KeychainError: Error {
+        case duplicateEntry
+        case unknown(OSStatus)
+    }
+    
+    static func save(service: String, account: String, password: Data) throws {
+        let query: [String: AnyObject] = [
+            kSecClass as String: kSecClassGenericPassword,
+            kSecAttrService as String: service as AnyObject,
+            kSecAttrAccount as String: account as AnyObject,
+            kSecValueData as String: password as AnyObject,
+        ]
+        
+        let status = SecItemAdd(query as CFDictionary, nil)
```

Comment:
Isn't this locking the user to a single password for a single `service` ever? I mean - no `Update` function present here. or remove